src/GameServer.js

```diff
@@ -385,20 +385,23 @@ GameServer.prototype.mainLoop = function() {
     this.time = local;
 
     if (!this.run) return;
+    
+    // The node & client updating mechanism is perfomance overhauled
+    // Nodes & clients will update periodically and never all on 25ms/50ms
+    // PlayerTracker and all Cell types have own internal update timer which is measured
+    setTimeout(this.updateClients.bind(this), 0);
+    setTimeout(this.moveTick.bind(this), 0);
```

Comment:
It's incorrect behavior, because CellUpdate message should be sent on each server tick. And it should be called each 40 milliseconds. It means that move tick and always should call update clients. 
If you call update clients asynchronously from move tick, you will not be able to get smooth movements on the client. It will be jerky and laggy.